src/library/io/mm-reader.cpp

```diff
@@ -569,7 +569,7 @@ clsparseSCsrMatrixfromFile(clsparseCsrMatrix* csrMatx, const char* filePath, cls
     MatrixMarketReader< cl_float > mm_reader;
     if( mm_reader.MMReadFormat( filePath ) )
         return clsparseInvalidFile;
-
+#if 0
```

Comment:
@kknox  Joseph is right, I disabled it to bypass the bug raised in #144. Just revert it back